client/tests/test.js

```diff
@@ -1,9 +1,23 @@
+var request = require('supertest')
```

Comment:
Look into this whole file. Possibly very old stuff. Didn't change anything recently.